### DISCUSSION TITLE
Create window without title bar that is draggable

### DISCUSSION THREAD

**Discussion by pablokentch** [microsoft/WindowsAppSDK#2962] [Q&A]
I'm converting an application made in WPF to WinUI 3 and Blazor

But I come across a problem to make my custom title bar made in blazor draggable

In WPF there is a `DragMove()` function

I defined in the creation of my application the following:

```
window.ExtendsContentIntoTitleBar = false;
Presenter.SetBorderAndTitleBar(false, false);
```

My blazor title bar I use:

```
<div class="bg-default w-full h-3 fixed" @onmousedown="@MoveWindow">
 ....
</div>

@code {
    private void MoveWindow()
    {
    
    }
}
```

Does anyone know how I can make draggable blazor title bar?

**Comment by castorix** [comment]
You can use Pointer events, like I did in this test [WinUI3_Transparent](https://github.com/castorix/WinUI3_Transparent)
(transparency does not work anymore since they added a background rectangle on the XAML Composition window...)

**Reply by pablokentch** [reply]
Is there any other way to do this in blazor?

**Reply by AlexanderBlackman** [reply]
> You can use Pointer events, like I did in this test [WinUI3_Transparent](https://github.com/castorix/WinUI3_Transparent) 

This is exactly what I'm looking for, thank you. 

<edit>  BTW, what is did you do to disable resizing from the bottom and sides?  When I reenable _presenter.IsResizable = true; I only can resize from the top of the window.
